src/main/java/com/spider/solomon/order/OrderItemDao.java

```diff
@@ -0,0 +1,31 @@
+package com.spider.solomon.order;
+
+import java.util.List;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import jakarta.annotation.Resource;
+
+@Repository
+public class OrderItemDao {
+	@Resource
+	JdbcTemplate jdbcTemplate;
+
+	public List<OrderItem> getOrderItemsByOrderId (int orderId) {
+		return jdbcTemplate.query("SELECT * FROM order_items WHERE order_id = ?", M, orderId);
```

Comment:
`M` could be a bit confusing -- could we rename to something like `ORDER_ITEM_MAPPER`?